10-inference.Rmd

```diff
@@ -83,124 +87,120 @@ The population consists of all studio apartment rentals in Vancouver, and the po
 
-```{r 11-example-proportions2, echo = TRUE, message = FALSE, warning = FALSE}
-virtual_box %>%
-  group_by(flavour) %>%
+airbnb %>%
   summarize(
-    n = n(),
-    proportion = n() / 10000
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
   )
 ```
 
-What would happen if we were to buy a box of 40 randomly-selected Timbits and count the number of chocolate Timbits (*i.e.,* take a random sample of size 40 from our Timbits population)? Let's use R to simulate this using our `virtual_box` population. We can do this using the `rep_sample_n` function from the `infer` package. The arguments
-of `rep_sample_n` are (1) the data frame (or tibble) to sample from, and (2) the size of the sample to take.
+```{r 11-population-parameter, echo = F}
+population_proportion <- airbnb %>%
+  summarize(
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
+  ) %>% 
+  pull()
+```
+
+From our data, we can see that the proportion of `Entire home/apt` listings in the data set is `r round(population_proportion,3)`. This value, `r round(population_proportion,3)`, is the *population parameter*. Remember, this parameter value is usually unknown in real data analysis problems.
+
+What would happen if we were to randomly select 40 listings and count the number of entire home/apartment listings (*i.e.,* take a random sample of size 40 from our population)? Let's use R to simulate this using our `airbnb` population. We can do this using the `rep_sample_n` function from the `infer` package. The arguments of `rep_sample_n` are (1) the data frame (or tibble) to sample from, and (2) the size of the sample to take.
 
 ```{r 11-example-proportions3, echo = TRUE, message = FALSE, warning = FALSE}
-set.seed(1)
-samples_1 <- rep_sample_n(tbl = virtual_box, size = 40)
-choc_sample_1 <- summarize(samples_1,
-  n = sum(flavour == "chocolate"),
-  prop = sum(flavour == "chocolate") / 40
+set.seed(123)
+sample_1 <- rep_sample_n(tbl = airbnb, size = 40)
+airbnb_sample_1 <- summarize(sample_1,
+  n = sum(room_type == "Entire home/apt"),
+  prop = sum(room_type == "Entire home/apt") / 40
 )
-choc_sample_1
+airbnb_sample_1
 ```
-Here we see that the proportion of chocolate Timbits in this random sample is
-`r round(choc_sample_1$prop,2)`. This value is our estimate &mdash; our best guess of our population parameter using this sample. Given that it is a single
+Here we see that the proportion of entire home/apartment listings in this random sample is
+`r round(airbnb_sample_1$prop,2)`. This value is our *estimate* &mdash; our best guess of our population parameter using this sample. Given that it is a single
 value that we are estimating, we often refer to it as a **point estimate**.
 
-Now imagine we took another random sample of 40 Timbits from the population. Do you
+Now imagine we took another random sample of 40 listings from the population. Do you
```

Comment:
Again, just another "what if"

instead, motivate like so:

"Our point estimate is 0.7. wow, that's close to our true population value. but remember, we just used a random sample of 40 to estimate it. keyword there is random -- if we were to take another random sample, we wouldn't get the same answer:"

[code and result]

"confirmed! we don't get the same answer. bummer. so that means that our point estimate might be unreliable! that is sampling variability etc etc"